webapp/channels/src/sass/routes/_settings.scss

```diff
@@ -267,6 +267,12 @@
                         line-height: 20px;
                     }
 
+                    .extraContentBeforeSettingList {
+                        display: inline-block;
```

Comment:
Shouldn't this be a `display: block` when the width is 100%?

Reply:
agreed, i changed it